man/kbmadm.1m

```diff
@@ -0,0 +1,249 @@
+.\"
+.\" This file and its contents are supplied under the terms of the
+.\" Common Development and Distribution License ("CDDL"), version 1.0.
+.\" You may only use this file in accordance with the terms of version
+.\" 1.0 of the CDDL.
+.\"
+.\" A full copy of the text of the CDDL should have accompanied this
+.\" source.  A copy of the CDDL is also available via the Internet at
+.\" http://www.illumos.org/license/CDDL.
+.\"
+.\"
+.\" Copyright 2020 Joyent, Inc.
+.\"
+.Dd January 16, 2020
+.Dt KBMADM 1M
+.Os
+.Sh NAME
+.Nm kbmadm
+.Nd Manage ebox-protected zfs dataset keys
+.Sh SYNOPSIS
+.Nm
+.Cm create-zpool
+.Op Fl g Ar guid
+.Op Fl t Ar template
+--
+.Ar zpool-create-args
+.Nm
+.Cm recover
+.Op Fl c Ar cfgnum
+.Ar dataset
+.Nm
+.Cm unlock
+.Op Fl r
+.Ar dataset
+.Nm
+.Cm recovery add
+.Op Fl f
+.Op Fl r Ar recovery_token
+.Fl t Ar template
+.Ar dataset
+.Nm
+.Cm recovery list
+.Op Fl p
+.Nm
+.Cm recovery activate
+.Ar dataset
+.Nm
+.Cm recovery cancel
+.Ar dataset
+.Sh DESCRIPTION
+.Nm
+is used to to manage
+.Xr kbmd 1M .
+.Sh SUBCOMMANDS
+.Bl -tag -width ""
+.\"
+.\" create-zpool
+.\"
+.It Xo
+.Nm
+.Cm create-zpool
+.Op Fl g Ar guid
+.Op Fl t Ar template
+.Ar dataset
+--
+.Ar zpool-create-args
+.Xc
+Create a new zpool with the root dataset encrypted.
+.Pp
+If
+.Ar guid
+is not specified, an uninitialized PIV token must be present on the system.
+.Pp
+The
+.Nm
+.Cm zpool-create
+subcommand will initialize the PIV token, call the
+.Sy register-pivtoken
+plugin method, and then create the zpool.
+.Bl -tag -width Fl
+.It Fl g Ar guid
+Do not initialize a PIV token.
+Instead, use the PIV token identified by
+.Ar guid
+to create the ebox.
+The PIV token must be inserted on the system at the time of the
+command invocation or it will fail.
+.It Fl t Ar template
+Read the recovery template from
+.Ar template .
+When initializing a PIV token, the
+.Sy register-pivtoken
+plugin method normally specifies the recovery configuration to use.
+Specifying this option will override any recovery configuration provided by the
+.Sy register-pivtoken
+plugin method.
+.Pp
+If the
+.Fl t Ar template
+option is not specified and either the
+.Fl g Ar guid
+option is also not specified, or the
+.Sy register-pivtoken
+plugin method does not specify a recovery configuration, the
+resulting ebox will not contain a recovery configuration.
+.It Ar zpool-create-args
```

Comment:
In such a case, loss of the primary token will result in permanent data loss.

Reply:
Added

Reply:
What you added is "loss of **a** primary token" rather than "loss of **the** primary token".  Your wording implies that more than one primary token may exist.  If multiple primary tokens exist, are all required?

Reply:
confirmed